integration-tests/tests/unbond_worker.rs

```diff
@@ -0,0 +1,98 @@
+use cosmwasm_std::{Addr, StdError};
+use integration_tests::contract::Contract;
+use std::collections::HashSet;
+
+pub mod test_utils;
+
+#[test]
+fn worker_should_be_in_collective_active_worker_set_of_all_supported_chains() {
+    let (protocol, ethereum, polygon, initial_workers, _) = test_utils::setup_test_case();
```

Comment:
not in scope for this PR, but something to think about: `setup_test_case` hides too much detail. I can't really see what this test does without digging three layers deep. We should remodel that, maybe with a builder pattern like so:
```
let setup = set_up_test_case()
    .with_workers(...)
    .with_chains(...)

```

Reply:
Ok sure. I will add a ticket for it to the backlog to later come back to this.